src/aioswitcher/device/tools.py

```diff
@@ -259,10 +269,13 @@ def get_light_discovery_packet_index(
     (based of device type and circuit number).
     """
     if device_type == DeviceType.RUNNER_S11:
-        if circuit_number in [0, 1]:
-            return circuit_number
-        else:
+        if circuit_number not in [0, 1]:
             raise ValueError("Invalid circuit number")
+        return circuit_number
+    if device_type == DeviceType.RUNNER_S12:
+        if circuit_number != 0:
+            raise ValueError("Invalid circuit number")
```

Comment:
Merge into one if using `and`

Reply:
`    if device_type == DeviceType.RUNNER_S11:
        if circuit_number not in [0, 1]:
            raise ValueError("Invalid circuit number")
        return circuit_number
    if device_type == DeviceType.RUNNER_S12:
        if circuit_number != 0:
            raise ValueError("Invalid circuit number")
        return 0

    raise ValueError("only devices that has lights are allowed")`

I cant make "and" here at it will break the logic.